app/pages/settings/email.jsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+
+class EmailSettingsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    console.log(props)
+    this.state = {
+      page: 1,
+      projects: [],
+      projectPreferences: []
+    };
+    this.handlePreferenceChange = this.handlePreferenceChange.bind(this);
+    this.nameOfPreference = this.nameOfPreference.bind(this);
```

Comment:
You only need to bind event handlers here. The other methods will have been automatically bound to the component context.